dashboard/src/shared/url.ts

```diff
@@ -23,13 +23,13 @@ export const api = {
     base: "api/chartsvc/v1",
     get: (id: string) => `${api.charts.base}/charts/${id}`,
     getReadme: (id: string, version: string) =>
-      `${api.charts.base}/assets/${id}/versions/${version}/README.md`,
+      `${api.charts.base}/assets/${id}/versions/${encodeURIComponent(version)}/README.md`,
     getValues: (id: string, version: string) =>
-      `${api.charts.base}/assets/${id}/versions/${version}/values.yaml`,
+      `${api.charts.base}/assets/${id}/versions/${encodeURIComponent(version)}/values.yaml`,
     getSchema: (id: string, version: string) =>
-      `${api.charts.base}/assets/${id}/versions/${version}/values.schema.json`,
+      `${api.charts.base}/assets/${id}/versions/${encodeURIComponent(version)}/values.schema.json`,
     getVersion: (id: string, version: string) =>
-      `${api.charts.base}/charts/${id}/versions/${version}`,
+      `${api.charts.base}/charts/${id}/versions/${encodeURIComponent(version)}`,
```

Comment:
Looking great. It would be great, but not required, if you could add a tests to show your change fixes the issue. For example, for `getChartVersion` you could add a test here:

https://github.com/kubeapps/kubeapps/blob/master/dashboard/src/actions/charts.test.tsx#L87

with a version with a tag (just copy the existing test and add the tag). You should find it fails without your above change, but passes with. Again, no problem if you don't have time to do so.

Reply:
No problem :) I've added test as you suggested. Seems good.